homeassistant/components/asuswrt/bridge.py

```diff
@@ -354,7 +362,7 @@ def __init__(self, conf: dict[str, Any], session: ClientSession) -> None:
         super().__init__(conf[CONF_HOST])
         # Get API configuration
         config = self._get_api_config()
-        self._api = self._get_api(conf, session, config)
+        self._api: AsusRouter = self._get_api(conf, session, config)
```

Comment:
Why isn't this defined on class level and typed there?
We shouldn't have to type it here.